backend/tests/fixtures/create_bruger_andersine.json

```diff
@@ -4,6 +4,7 @@
             {
                 "brugernavn": "Andersine And",
                 "brugervendtnoegle": "andersineand",
+                "integrationsdata": "{\"von-and-løn-id\": \"2468\"}",
```

Comment:
Not your fault, but JSON-in-JSON strikes me as quite horrible 😞

Reply:
As Dan writes below, there's not actually any reason to have the JSON escaped. It can be nested so that it's treated completely like nested JSON, even if MO is doing nothing but carry it around. Nothing is gained by having it as a string, as far as I can see.

Or is there some complication I haven't thought of?

Reply:
Yes - Json fields was a nogo - but convrint on the way in and out was.